File: intermediates/exo2.R
library(readr)
library(dplyr)
library(stringr)
library(sf)
library(httr2)
library(utils)

source("correction/R/import_data.R")
source("correction/R/create_data_list.R")
source("correction/R/clean_dataframe.R")


# Load data ----------------------------------
urls <- create_data_list("./sources.yml")


pax_apt_all <- import_airport_data(download_and_unzip(urls$airports))
pax_cie_all <- import_compagnies_data(download_and_unzip(urls$compagnies))
pax_lsn_all <- import_liaisons_data(download_and_unzip(urls$liaisons))

airports_location <- st_read(urls$geojson$airport)

File: intermediates/exo2.py
import pandas as pd
import geopandas as gpd

import src.import_data as sid
from src.create_data_list import create_data_list

# Load data ----------------------------------
urls = create_data_list("./sources.yml")


pax_apt_all = sid.import_airport_data(sid.download_and_unzip(urls['airports']))
pax_cie_all = sid.import_airport_data(sid.download_and_unzip(urls['compagnies']))
pax_lsn_all = sid.import_airport_data(sid.download_and_unzip(urls['liaisons']))


airports_location = gpd.read_file(
    urls['geojson']['airport']
)
